official/resnet/keras/keras_common.py

```diff
@@ -25,8 +25,9 @@
 # pylint: disable=g-bad-import-order
 from absl import flags
 import tensorflow as tf
-from tensorflow.python.keras.optimizer_v2 import (gradient_descent as
-                                                  gradient_descent_v2)
+
+from tensorflow.core.protobuf import rewriter_config_pb2
+from tensorflow.python.keras.optimizer_v2 import gradient_descent as gradient_descent_v2
```

Comment:
Flagged as too long by linter, you can change it back to how we had it before above.

Reply:
Done. Changed it back to original.